client/my-sites/site-settings/form-general.jsx

```diff
@@ -63,21 +63,7 @@ import wrapSettingsForm from './wrap-settings-form';
 
 export class SiteSettingsFormGeneral extends Component {
 	componentDidMount() {
-		// Wait for page.js to update the URL, then see if we are linking
-		// directly to a section of this page.
-		setTimeout( () => {
-			if ( ! window || ! window.location ) {
-				// This code breaks everything in the tests (they hang with no
-				// error message).
-				return;
-			}
-			const hash = window.location.hash;
-			const el = hash && document.getElementById( hash.substring( 1 ) );
-			if ( hash && el ) {
-				const y = el.offsetTop - document.getElementById( 'header' ).offsetHeight - 15;
-				scrollTo( { y } );
-			}
-		} );
+		setTimeout( () => scrollToAnchor( { offset: 15 } ) );
```

Comment:
Would this be worth including in the Testing Instructions?

Reply:
I already included it there as the first link to test `/settings/general/SITE_URL#site-privacy-settings`.